fluent/typings/index.d.ts

```diff
@@ -0,0 +1,6398 @@
+// Generated by `npm run typings`
```

Comment:
Should  this file be checked into git or generated as build time?

Reply:
We could, but `npm run test-generated` generates a copy and compares it to this one to assert that the declarations haven't changed unexpectedly.